### ISSUE TITLE
upgrade the Polish translation

### ISSUE THREAD

**Issue by paxcoder** [losnoco/Cog#314]
So, I managed to translate 11% of cog's ui into Polish.
I am waiting to see the new version.

**Comment by kode54** [comment]
I imported the rest of your previous .strings files translation, and fixed up a few places where you either put a string in the wrong field (down the list maybe two or three places from where you meant to fill it in), or a few where you translated token names. I also fixed a few places with either stray linefeeds, or doubled spaces.

And, subject to your revisions, I attempted to translate a few things I thought you would appreciate for now, like the path exception dialog calling English "valid", a Polish word close to the English word "operational", since the direct translation for "valid" seemed to not fit there in the context. Again, I seek your approval.

With mostly your own changes imported, and a few things I attempted as well, we're up to about 36% Polish completion. I look forward to your future contributions to the Lokalise site, as I'll be sticking to letting you do your thing from now on, except I'll be proofreading a bit to make sure there are no further string misplacements.

A bit of a hint for future reference, the site has a bit of a bug with scrolling itself randomly when activating or typing into the Editor boxes. I really wish it didn't have bugs like this.
